packages/editor/src/lib/components/DefaultErrorFallback.tsx

```diff
@@ -40,11 +40,11 @@ export const DefaultErrorFallback: TLErrorFallback = ({ error, app }) => {
 		},
 		[app]
 	)
-	const [isDarkMode, setIsDarkMode] = useState<null | boolean>(null)
```

Comment:
There is some unrelated consistency changes here. For our editor, we had both `isBoolean / setIsBoolean` and `isBoolean / setBoolean`, and I consolidated on `isBoolean / setBoolean` (for example, `isDarkMode` / `setDarkMode`). I've here also rolled this out to other usage in the app.